frontend/src/app/homepage.cljs

```diff
@@ -79,6 +93,29 @@
      (render-navigation-item "Upload" "#upload")
      (render-navigation-item "Container" "#container")]])
 
+(defn progress-width []
+  (min
+    (*
+      (float
+        (/
+          (:total_reports @backend-stats)
+    @report-target)) 100) 100)
+)
+
+(defn render-stats []
+  [:div {:id "progressbar"}
+    [:div
+      [:h5 "Are we there yet?"]
+    ]
+    [:div {
+      :id "progress"
+      :style {:width (str (progress-width) "%")}
+      }
+      [:p (progress-width) "%"]
+    ]
+  ]
+)
```

Comment:
In Lisp languages, it is standard practice to join all ending brackets on one like. The same way you did in the `fetch-stats-backend` function.